packages/context-menu/test/items.test.js

```diff
@@ -506,24 +506,36 @@ describe('items', () => {
       `);
       rootMenu.openOn = menuOpenEvent;
       target = rootMenu.firstElementChild;
+
+      const itemWithTheme = document.createElement('span');
+      itemWithTheme.textContent = 'Item with theme';
+      itemWithTheme.setAttribute('theme', 'bar');
+
       rootMenu.items = [
         {
           text: 'foo-0',
-          children: [{ text: 'foo-0-0' }, { text: 'foo-0-1', children: [{ text: 'foo-0-1-0' }] }]
+          children: [
+            { text: 'foo-0-0' },
+            {
+              text: 'foo-0-1',
+              children: [{ text: 'foo-0-1-0' }]
+            },
+            { component: itemWithTheme }
+          ]
         },
         { text: 'foo-1' }
       ];
       open();
-      await nextFrame();
+      await nextRender();
```

Comment:
Changed to use `nextRender()` which seems to be needed to make all the nested menus rendered properly when debugging (otherwise the elements are in the DOM but not visible).